packages/storefront-lwc/src/modules/commerce/router/router.js

```diff
@@ -11,7 +12,10 @@ const router = new UniversalRouter(
         {
             path: '/search/:query',
             action: location => {
-                return `<commerce-product-search-results query="${location.params.query}"></commerce-product-search-results>`;
+                var query = `${location.params.query}`;
+                query = xss(query);
+                console.log('router: ', query);
```

Comment:
There should be no `console.log`. Also it would be simpler  just to do `query="${xss(location.params.query)}"`